src/js/main.js

```diff
@@ -52,10 +52,15 @@ Site.Menu = {
 
   onItemClick: function(item, event) {
     var _this = this;
+    var href = $(item).children('a').attr('href');
+    var hash = href.substr(1);
 
     $('.header-menu-active').removeClass('header-menu-active');
     $(item).addClass('header-menu-active');
-  }
+
+    $('.page-content').removeClass('page-content-active');
+    $('.page-content[data-slug="' + hash + '"]').addClass('page-content-active');
```

Comment:
I'm cool with this approach and it works well in practice. But I think the client asked for a full scroll in the design comments so need to clarify that